data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/impl/ReceiverVerticle.java

```diff
@@ -82,23 +81,35 @@ public class ReceiverVerticle extends AbstractVerticle implements Handler<HttpSe
     private MessageConsumer<Object> messageConsumer;
     private IngressProducerReconcilableStore ingressProducerStore;
 
+    private SecretWatcher secretWatcher;
+
+    public HttpServerOptions getHttpsServerOptions() {
+        return httpsServerOptions;
+    }
```

Comment:
Can we move this method after the constructor? Ideally, we place it at the bottom of the class since it's a minor method